config/locales/view.en.yml

```diff
@@ -57,3 +58,11 @@ en:
       column_pvp: "PVP Plans"
       column_enrollments: "Active Enrollments"
       column_products: "Products"
+    carrier:
+      title: "%{year} %{carrier_name}"
```

Comment:
in the mockup this ends with "Plans", so probably should here?

Reply:
Good catch. Thank you